source/sdk/lang/types.ooc

```diff
@@ -15,10 +15,8 @@ Object: abstract class {
     __destroy__: func {}
 
     free: virtual func {
-      version(!gc) {
-        this __destroy__()
-        gc_free(this)
-      }
+      this __destroy__()
```

Comment:
Ah, yes, `__destroy__` doesn't do anything at all here. Will fix in a future PR.